src/Sentry.Unity/SentryOptionsUtility.cs

```diff
@@ -11,6 +11,7 @@ public static void SetDefaults(SentryUnityOptions options, IApplication? applica
 
             // 'Optimal' and 'Fastest' don't work on IL2CPP. Forcing 'NoCompression'.
             options.RequestBodyCompressionLevel = CompressionLevelWithAuto.NoCompression;
+            options.IsEnvironmentUser = false;
```

Comment:
We could test that a captured event doesn't have `evt.User.UserName` equal to `Environment.MachineName` with the default `SentryUnityOptions`

Reply:
As discussed we're checking for Environment.UserName. See
https://github.com/getsentry/sentry-dotnet/blob/0f4358d1f37f664d7e144d33f6db5db842180726/src/Sentry/Internal/Enricher.cs#L80